src/languages/en.js

```diff
@@ -522,6 +522,8 @@ export default {
         tryAgain: 'Try again',
         verifyIdentity: 'Verify identity',
         genericError: 'There was an error while processing this step. Please try again.',
+        cameraRequestTitle: '"New Expensify" Would Like to Access the Camera',
```

Comment:
We will also need to ask Marketing team to confirm these translations. I will try to take care of it in the OG issue

Reply:
Thanks for the follow-up. I will wait until getting these official messages.

Reply:
@phivh Thank you for understanding, this might be a bit longer than usual given the holidays. Apologies for the delay.

Reply:
@mountiny I see, no problem. This is also a chance for me to get more understanding of the process.

Reply:
@phivh Could you actually use the key `` we have couple lines above here? https://github.com/Expensify/App/blob/bec6d3aa09d153d87c68e820fc5b456736c40458/src/languages/en.js#L514

Replate this line with the aforementioned line and swap the translations keys where you have used it to `cameraPermissionsNotGranted`. And remove this standalone key. I have checked and it seems it was not used anywhere.